packages/flutter/lib/src/widgets/nested_scroll_view.dart

```diff
@@ -855,7 +855,10 @@ class _NestedScrollCoordinator implements ScrollActivityDelegate, ScrollHoldCont
         position.maxScrollExtent - position.minScrollExtent,
```

Comment:
In fact, `maxInnerExtent` is not the total extension of the inner scrollable, but the maximum scrolling space inside, right?